src/main/kotlin/org/rust/ide/actions/mover/RsAbstractUpDownMover.kt

```diff
@@ -76,19 +63,62 @@ abstract class RsAbstractUpDownMover : StatementUpDownMover() {
             }
             element = parent
         }
+        return element
+    }
 
-        if (element != null) {
-            info.toMove2 = LineRange(element)
+    private fun checkContainerBoundaries(element: PsiElement?, collectedElement: PsiElement): Boolean {
```

Comment:
Looks like this could be `element: PsiElement?`

In general, functions which take a nullable parameter and do nothing if the parameter is null are code smell: it's usually better to push null-checks to the caller (and to avoid null checks at all if possible, by threading non-null types).